modules/alloy/src/main/ts/ephox/alloy/dragging/dragndrop/DropEvent.ts

```diff
@@ -12,7 +11,8 @@ export interface DropEvent extends SimulatedEvent<EventArgs<DragEvent>> {
 
 export const createDropEventDetails = (config: DroppingConfig, event: SimulatedEvent<EventArgs<DragEvent>>): DropEvent => {
   const rawEvent = event.event().raw();
-  const transfer = rawEvent.dataTransfer;
+  // TODO: Should this handle if dataTransfer is null?
+  const transfer = rawEvent.dataTransfer as DataTransfer;
```

Comment:
This was something the new types showed up as well. We weren't handling this previously either, so this is probably okay?